README.md

```diff
@@ -233,7 +237,7 @@ Indentation types are:
 * `:inner` -
   two character indentation applied to form arguments at a depth
   relative to a form symbol
-  
```

Comment:
Let's remove this whitespace cleanup from the commit, so we have a clean diff.

Reply:
Can it be a different commit then?

Reply:
It should ideally be a separate PR, so we can look at the diff of this PR and see only the changes needed to implement it.